querydsl-sql/src/main/java/com/querydsl/sql/SQLListeners.java

```diff
@@ -245,4 +245,9 @@ public void exception(final SQLListenerContext context) {
             listener.exception(context);
         }
     }
+
+    public Set<SQLDetailedListener> getListeners() {
```

Comment:
Does this specifically need to be a `Set`?
Just wondering if you had plans with this.